mlflow/models/evaluation/base.py

```diff
@@ -1307,26 +1308,36 @@ def evaluate(
                         def fn(model_input):
                             return model.predict(model_input)
 
+        If omitted, it indicates a static dataset will be used for evaluation instead of a model.
+        In this case, the ``data`` argument must be a Pandas DataFrame that contains model outputs,
+        and the ``prediction`` argument must be the name of the column in ``data`` that contains
+        model outputs.
+
     :param data: One of the following:
 
                  - A numpy array or list of evaluation features, excluding labels.
 
-                 - A Pandas DataFrame or Spark DataFrame, containing evaluation features and
-                   labels. If ``feature_names`` argument not specified, all columns are regarded
-                   as feature columns. Otherwise, only column names present in ``feature_names``
-                   are regarded as feature columns. If it is Spark DataFrame, only the first 10000
-                   rows in the Spark DataFrame will be used as evaluation data.
+                 - A Pandas DataFrame or Spark DataFrame, containing evaluation features,
+                   labels, and optionally model outputs. If ``feature_names`` argument not
+                   specified, all columns except for the label column and model_output column
+                   are regarded as feature columns. Otherwise, only column names present in
+                   ``feature_names`` are regarded as feature columns.
+                   If it is Spark DataFrame, only the first 10000 rows in the Spark DataFrame
+                   will be used as evaluation data.
 
-                 - A :py:class`mlflow.data.dataset.Dataset` instance containing evaluation features
-                   and labels.
+                 - A :py:class:`mlflow.data.dataset.Dataset` instance containing evaluation
+                   features and labels.
```

Comment:
thanks for catching this!

Reply:
I moved the fix to here https://github.com/mlflow/mlflow/pull/9926